coqlspclient/coq_structs.py

```diff
@@ -90,12 +90,41 @@ def __hash__(self) -> int:
         return hash(self.text)
 
 
-class ProofStep(Step):
-    def __init__(self, step: Step, goals: GoalAnswer, context: List[Term]):
-        super().__init__(step.text, step.ast)
-        self.goals = goals
+class ProofStep:
```

Comment:
Why is this no longer a Step?

Reply:
Because we want to update the Steps and change the respective ProofSteps. If a ProofStep has a Step instead of being one, we are able to do the changes related to the Steps in the CoqFile (namely ranges when we add or delete steps) and the changes related to the ProofStep on the ProofFile.